bigframes/_config/__init__.py

```diff
@@ -109,6 +128,11 @@ def is_bigquery_thread_local(self) -> bool:
 
         A thread-local session can be started by using
         `with bigframes.option_context("bigquery.some_option", "some-value"):`.
+
+        Returns:
+            bool:
+              A boolean value, where a value is True if a thread-local session
```

Comment:
Same here and the line below (2 more spaces)

Reply:
Done.